content/en/cosign/installation.md

```diff
@@ -4,9 +4,9 @@ category: "Cosign"
 position: 102
 ---
 
-## With Go 1.19+
+## With Go 1.19+ 
 
-If you have Go 1.19+, you can directly install Cosign by running:
+If you have Go 1.19+, you can directly install Cosign by downloading the cosign binary and running:
```

Comment:
In terms of branding, we've been title casing Cosign unless it's the command `cosign`.

```suggestion
If you have Go 1.19+, you can directly install Cosign by downloading the Cosign binary and running:
```